ssl-config-akka/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala

```diff
@@ -78,6 +72,10 @@ final class AkkaSSLConfig(system: ExtendedActorSystem) extends Extension {
     // TODO ...
   }
 
+  /** Prepare the passed in engine for a new session negotiation*/
+  def negotiateFirstNewSession(rawSslParameters: Option[SSLParameters]): NegotiateNewSession =
+    NegotiateNewSession(config.enabledCipherSuites, config.enabledProtocols, config.sslParametersConfig.clientAuth, rawSslParameters)
```

Comment:
Previously this was provided by HttpsContext in Akka HTTP. 
It was `firstSession` there.

Thanks to moving it here we are able to slim down HttpsContext to just SSLContext, as all the config can be taken from the config.